src/components/Personalization/actions/prependHtml.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { showElements } from "../flicker";
+
+const DIV_TAG = "DIV";
+
+const createFragment = content => {
+  const result = document.createElement(DIV_TAG);
+  result.innerHTML = content;
+
+  return result;
+};
+
+const prependHtml = (container, content) => {
+  const fragment = createFragment(content);
+  const elements = [].slice.call(fragment.children);
+
+  elements.forEach(element => {
+    container.insertBefore(element, container.firstElementChild);
+  });
```

Comment:
If `fragment.children` has more than one child, wouldn't the children end up unexpectedly in reverse order?

Reply:
Good catch, adjusted and added proper unit test to account for this.